appveyor.yml

```diff
@@ -1,11 +1,12 @@
 environment:
-  LEATHERMAN_VERSION: 0.4.2
+  LEATHERMAN_VERSION: 0.9.0
 install:
   - git submodule update --init --recursive
 
   - choco install -y mingw-w64 -Version 4.8.3 -source https://www.myget.org/F/puppetlabs
   - choco install -y cmake -Version 3.2.2 -source https://www.myget.org/F/puppetlabs
-  - SET PATH=C:\tools\mingw64\bin;%PATH%
+  - choco install -y gettext -Version 0.19.6 -source https://www.myget.org/F/puppetlabs
```

Comment:
You'll also want to update .travis_target.sh to install a recent version of gettext, as in https://github.com/puppetlabs/leatherman/blob/master/scripts/travis_target.sh#L21-L25